ztp/gitops-subscriptions/argocd/example/optional-extra-manifest/ipsec/README.md

```diff
@@ -57,6 +59,8 @@ Run the build script to create the MachineConfig:
 
 `./build.sh`
 
-Apply the MachineConfig:
+Include the `99-ipsec-master-endpoint-config.yaml` in the additional install-time manifests folder set in SiteConfig.
```

Comment:
Lets rephrase these to something like this:
(BTW is the folder install-time or sno-extra-manifest?)

To deploy with ZTP, include `99-ipsec-master-endpoint-config.yaml` in the additional install-time manifests folder set in SiteConfig.

Or optionally, it can be manually applied with: